packages/trader/src/App/Components/Layout/Header/platform-dropdown.jsx

```diff
@@ -0,0 +1,47 @@
+import classNames     from 'classnames';
+import PropTypes      from 'prop-types';
+import React          from 'react';
+import ReactDOM       from 'react-dom';
+import { BinaryLink } from 'App/Components/Routes';
+import Icon           from 'Assets/icon.jsx';
+import 'Sass/app/_common/components/platform-dropdown.scss';
+
+const PlatformDropdown = ({
+    platform_config,
+    handleClick,
+}) => ReactDOM.createPortal(
+    <div className='platform_dropdown' onClick={handleClick}>
+        <div className='platform_dropdown__list'>
+            {platform_config.map((platform, idx) => {
+                const is_bot = /^\/bot/.test(location.pathname);
```

Comment:
this will always looping the test. you can just run it once before creating the portal